src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -966,7 +991,7 @@ function discoverController(
         schema: 'segment',
         params: {
           field: $scope.opts.timefield,
-          interval: $state.interval,
+          interval: $scope.state.interval,
```

Comment:
Why not use timefilter.getInterval here?

Reply:
because its the interval of the histogram:

![Bildschirmfoto 2020-03-05 um 11 50 36](https://user-images.githubusercontent.com/463851/75974798-af490800-5ed7-11ea-80b4-4befbb38fb53.png)

Reply:
Maybe then adjust the name to `histogramInterval`?

Reply:
that would be much clearer, but would also mean to change the name of a state variable, which would need a migration step. I think a comment would also help, don't you think?

Reply:
No biggie, we can handle it another time.